libs/core/src/lib/list/list-item/list-item.component.ts

```diff
@@ -161,6 +169,9 @@ export class ListItemComponent extends ListFocusItem implements AfterContentInit
                 this.radio.labelClicked(event);
                 this._muteEvent(event);
             }
+            if (this.link && this._url) {
+                window.open(this._url, '_self');
```

Comment:
maybe we need to add `@Input` for target to `list-link.directive` so that the user can define how this link will open?

Reply:
Yes, the application developer needs to be able to choose between this, Angular router etc